snapshot-labs/sx-monorepo
# feat: add leaderboard for onchain spaces

### Summary



Closes: #250

![Screenshot 2024-04-17 at 19 04 17](https://github.com/snapshot-labs/sx-monorepo/assets/495709/cec629d0-4986-4559-a340-a41aab6fc160)


### How to test

1. Go to an onchain spaces
2. In the sidebar, click on the new "Leaderboard" menu
3. It should show all the space's voters and proposals authors, sorted by descending votes count  
4. You should be able to load more users with the infinite scroll
5. You should be able to sort by votes and proposals count

## TODO

- [x] A little code cleanup before sending for review
